scripts/generate-import.sh

```diff
@@ -1,16 +1,26 @@
-#!/usr/bin/env bash
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p yq jq curl
 set -eou pipefail
 [[ ! -z ${DEBUG:-} ]] && set -x
 
+TOKEN="Authorization: token $(yq -r '.["github.com"][0].oauth_token' ~/.config/hub)"
```

Comment:
We all probably have hub, but I'd like to see either a fallback or at least a meaningful error message.